rocketpy/sensors/barometer.py

```diff
@@ -150,16 +150,12 @@ def measure(self, time, **kwargs):
                 Derivative of the state vector of the rocket.
             - relative_position : np.array
                 Position of the sensor relative to the rocket center of mass.
-            - gravity : float
-                Gravitational acceleration in m/s^2.
-            - pressure : Function
-                Atmospheric pressure profile as a function of altitude in Pa.
-            - elevation : float
-                Elevation of the launch site in meters.
+            - environment : Environment
+                Environment object containing the atmospheric conditions.
```

Comment:
Receiving an Environment? At a first glance, this seems odd to me. Let me give a try.